README.md

```diff
@@ -350,3 +351,18 @@ $ wdl parse examples/ex2.wdl
         (RawCommand:
 ...
 ```
+
+# Converting To DOT
+
+A wdl file can be converted to the dot format in order to be able to visualize the pipeline as a graph. For example:
+
+```
+$ python scripts/wdl2dot.py -i hello.wdl -o hello.dot
```

Comment:
This will also have the added benefit of making `wdl2dot` available to people who install PyWDL via `pip install pywdl`